### PR TITLE
[IOPID-958] Added FL OPT-IN Feature Flag

### PR DESCRIPTION
**Pull request by sabontech**
## Short description
Just updated `package.json` and added a standard configuration to `src/payloads/backend.ts`

### CODE REVIEW

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Review by Vangaorth — CHANGES_REQUESTED** [review]
See comment below

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "api_cgn": "https://raw.githubusercontent.com/pagopa/io-backend/v13.10.1-RELEASE/api_cgn.yaml",
   "api_cgn_merchants": "https://raw.githubusercontent.com/pagopa/io-backend/v13.10.1-RELEASE/api_cgn_operator_search.yaml",
   "api_cgn_geo": "https://raw.githubusercontent.com/pagopa/io-backend/here_geoapi_integration/api_geo.yaml",
-  "content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.27/definitions.yml",
+  "content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.28/definitions.yml",
```
Comment (Vangaorth) [inline package.json:13]: Should not we keep every io-services-metadata version aligned?

**Review by Vangaorth — CHANGES_REQUESTED** [review]

**Review by Vangaorth — APPROVED** [review]

**Review by Vangaorth — APPROVED** [review]
LGTM

**Pull request merged**